### ISSUE TITLE
dark search with FF

### ISSUE THREAD

**Issue by matteomarchesi** [Jazqa/adwaita-tweaks#18]
Hi, with the "dark" version and with FiireFox I got dark search field.
I'm on Ubuntu 16.04 with Gnome 3.20.

![image](https://user-images.githubusercontent.com/19588657/33495923-8d4ef686-d6c8-11e7-88d7-4fde28d95760.png)

**Comment by Jazqa** [comment]
As far as I know, this used to happen with most dark themes. I haven't used Firefox in a while, is this only the case with Adwaita Tweaks or does it happen with the regular Adwaita as well?

**Comment by matteomarchesi** [comment]
yes, it happens with other dark themes too. So it's not that dark theme makers are lazy and do not remember to make it clear and readable ;-) LOL
Any way to fix it?
